timers.go

```diff
@@ -19,6 +19,14 @@ type Timer struct {
 	Timestamp    Nanotime    // Last time value was updated
 	Hostname     string      // Hostname of the source of the metric
 	Tags         Tags        // The tags for the timer
+
+	// Map bounds to count of measures seen in that bucket.
+	// This map only non-empty if the metric specifies percentile aggregation in its tags.
+	Histogram map[HistogramThreshold]int
+}
+
+type HistogramThreshold struct {
```

Comment:
What's the value in having a struct instead of just a float?

Reply:
probably none, will get rid of that

Reply:
wanted to use
```golang
type HistogramThreshold float64
```
because I think it helps with readability, but then I can't compare `float64`s with it. Any tip on that?

Reply:
`x < HistogramThreshold(y)` or `float64(x) < y` should work.